src/bytecode/instr.py

```diff
@@ -22,46 +23,46 @@
 # Instructions relying on a bit to modify its behavior.
 # The lowest bit is used to encode custom behavior.
 BITFLAG_OPCODES = (
-    (
+    {
         _opcode.opmap["BUILD_INTERPOLATION"],
         _opcode.opmap["LOAD_GLOBAL"],
         _opcode.opmap["LOAD_ATTR"],
-    )
+    }
     if PY314
     else (
-        (_opcode.opmap["LOAD_GLOBAL"], _opcode.opmap["LOAD_ATTR"])
+        {_opcode.opmap["LOAD_GLOBAL"], _opcode.opmap["LOAD_ATTR"]}
         if PY312
-        else ((_opcode.opmap["LOAD_GLOBAL"],) if PY311 else ())
+        else ({_opcode.opmap["LOAD_GLOBAL"]} if PY311 else set())
     )
 )
 
-BITFLAG2_OPCODES = (_opcode.opmap["LOAD_SUPER_ATTR"],) if PY312 else ()
+BITFLAG2_OPCODES = {_opcode.opmap["LOAD_SUPER_ATTR"]} if PY312 else set()
```

Comment:
Out of curiosity do you have a micro-benchmark for this. It is not obvious to me that for a single element a set is faster ?

Reply:
In fact I've got a pretty decent improvement when turning `HASJREL` etc... into sets. I'll post some benchmarking notes in the description.

Reply:
Just throwing an idea there: when the library is imported it is "locked" to the version of Python it is running under. There is no reason for a particular version of this library to support a range of Python interpreters. The only downside that I can see from migrating to a scenario where a release supports one specific Python release is that it would require some awkward dependency specification in other projects. However, in a [project that I work on](https://github.com/DataDog/dd-trace-py/blob/348e549220114e0daf15f3592579eed92d3746cd/pyproject.toml#L38-L42) this is already the case anyway. For example

```
bytecode~=0.17.0; python_version~=3.14.0
bytecode~=0.16.0; python_version~=3.13.0
...
```